src/common/fs_file.h

```diff
@@ -46,42 +47,31 @@ class File {
         return read(data.data(), data.size() * sizeof(T));
     }
 
-    bool isOpen() const {
-        return m_file != nullptr;
-    }
+    bool isOpen() const { return m_file != nullptr; }
 
     const char* getOpenMode(OpenMode mode) const {
         switch (mode) {
-            case OpenMode::Read:
-                return "rb";
-            case OpenMode::Write:
-                return "wb";
-            case OpenMode::ReadWrite:
-                return "r+b";
-            default:
-                return "r";
+            case OpenMode::Read: return "rb";
+            case OpenMode::Write: return "wb";
+            case OpenMode::ReadWrite: return "r+b";
+            default: return "r";
         }
     }
 
     int getSeekMode(SeekMode mode) const {
         switch (mode) {
-            case SeekMode::Set:
-                return SEEK_SET;
-            case SeekMode::Cur:
-                return SEEK_CUR;
-            case SeekMode::End:
-                return SEEK_END;
-            default:
-                return SEEK_SET;
+            case SeekMode::Set: return SEEK_SET;
+            case SeekMode::Cur: return SEEK_CUR;
+            case SeekMode::End: return SEEK_END;
+            default: return SEEK_SET;
         }
     }
 
-    [[nodiscard]] std::FILE* fileDescr() const {
-        return m_file;
-    }
+    [[nodiscard]] std::FILE* fileDescr() const { return m_file; }
+    static std::vector<DirEntry> getDirectoryEntries(const std::string& path);
 
   private:
     std::FILE* m_file{};
 };
 
-} // namespace Common::FS
+}  // namespace Common::FS
```

Comment:
```suggestion
} // namespace Common::FS
```
There should be only 1 space in namespace comment